### PR TITLE
Feature/add space prop channel value

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes PXBLUE-2442.

#### Changes proposed in this Pull Request:

- [X] Add unitSpace prop with `auto`, `show`, `hide`. Default `auto`.
- [X] Add knob in `with full config` storybook
- [X] Update testcases.
- [X] Update changelog

#### To Test:
- Storybook URL
http://localhost:6006/?path=/story/components-channel-value--with-full-config

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+### Added
+## 6.0.0 (Unreleased)
+-   `unitSpace` prop to `<ChannelValue>` to manage spacing between the value and units.
```
Comment (huayunh) [inline CHANGELOG.md:5]: Should write the changlog in Angular’s terminology …

**Comment by jeffvg** [comment]
should the percent symbol work with unitSpace?

image

File: CHANGELOG.md (hunk 1 above)
Comment (ektasawant) [reply CHANGELOG.md]: Updated readme file.

**Comment by emclaug2** [comment]
Since this is going to be a breaking change, instead of merging this into the dev branch directly, I'd merge this into a separate branch until we're ready to publish our next major version.  I just created a release/6.0.0 branch, could you update this PR to merge into that. 

Right now the release/6.0.0 branch based off of the dev branch so there shouldn't be any issues merging.

**Pull request closed without merging**